gocd/generated-pipelines/relay-next-monitor.yaml

```diff
@@ -7,8 +7,8 @@ pipelines:
     group: relay-next
     lock_behavior: unlockWhenFinished
     materials:
-      deploy-relay-next-us-pipeline-complete:
-        pipeline: deploy-relay-next-us
+      deploy-relay-next-pipeline-complete:
+        pipeline: deploy-relay-next
```

Comment:
also here, does this come from `libs`?

Reply:
Yeah, as mentioned in the PR description, the order of the pipedream is changed and this is how GoCD manages the ordering - via dependencies on pipelines.